examples/custom/prerequisites.tf

```diff
@@ -10,8 +10,7 @@ provider "aws" {
   skip_region_validation      = true
   skip_credentials_validation = true
   skip_get_ec2_platforms      = true
-  skip_metadata_api_check     = true
```

Comment:
I'm not sure about this one because it's under `examples/`. I can undo this change if you think this has nothing to do with fixing this issue.

Reply:
lgtm